Counter/app/build.gradle

```diff
@@ -16,17 +16,16 @@
 
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
-apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion "29.0.3"
+    compileSdkVersion 31
+    buildToolsVersion "30.0.3"
```

Comment:
I think we can just take this out now (https://stackoverflow.com/a/39239240/2639539)